### DISCUSSION TITLE
Unexpected token CloseCurlyBracket at [project]/styles/globals.css:147:1

### DISCUSSION THREAD

**Discussion by matcastaneda** [tailwindlabs/tailwindcss#14729] [Help]
![image](https://github.com/user-attachments/assets/b0782fba-3196-4cc7-ad71-ae86c3a65fce)

**Comment by gabrielvfonseca** [comment]
I am with the same error. Don't know how to fix it

**Reply by gabrielvfonseca** [reply]
Ok, the error just solved itself. 
At first, I tried deleting all the dependencies and was forced to remove the cache.
It didn't work, so I repeated the process and then restarted my computer.
After that, run `ppm i`, and it was all working fine.
